node/src/database/rocksdb.rs

```diff
@@ -356,15 +407,19 @@ impl<'db, DB: DBAccess> Mempool for DBTransaction<'db, DB> {
     }
 
     fn get_any_nullifier_exists(&self, nullifiers: Vec<[u8; 32]>) -> bool {
-        nullifiers
-            .into_iter()
-            .all(|n| self.snapshot.get_cf(self.nullifiers_cf, n).is_ok())
+        nullifiers.into_iter().all(|n| {
```

Comment:
Should this be `any` right?
```suggestion
        nullifiers.into_iter().any(|n| {
```

Reply:
Good catch!